### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by suretrust**
Add styling to the app through CSS and props.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
 class Button extends React.Component {
   render() {
-    return <button>{this.props.name}</button>;
+    const wide = this.props.wide ? 'fifty-percent' : '';
+    const color = this.props.color || '';
```
Comment (SunnyAsar) [inline src/components/Button.js]: Hey, if there is no prop sent, the color should be set from default props, we had that in the last milestone, what happened?
- you have css set for some button already let's not do that. use default props.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @suretrust 👋 
### Project looking good.
let's just make sure our default styles come from the component not from css.

**Please request another review with the update**
Happy Coding.

File: src/components/Button.js (hunk 1 above)
Comment (suretrust) [reply src/components/Button.js]: Thanks so much for pointing that out. Now amended.

**Comment by suretrust** [comment]
Amended as suggested. Kindly review.

**Review by BalintMendli — APPROVED** [review]
Hi @suretrust ,

All good now.

### 👍 

Let's move to the next milestone.

**Pull request merged**